components/baseline/setup.ftl

```diff
@@ -55,7 +55,7 @@
     [#local baselineComponentIds = getBaselineComponentIds(
       baselineLinks,
       AZURE_CMK_RESOURCE_TYPE,
-      AZURE_SSH_PRIVATE_KEY_RESOURCE_TYPE,
+      AZURE_KEYVAULT_SECRET_RESOURCE_TYPE,
```

Comment:
This one shouldn't be a resource type it should be the id of the ssh key pair resource `vmKeyPair`

Reply:
To make the distinction between the id vs type of cmk I would make it "cmk" as well

Reply:
This has been addressed in 0be0928